WordPress/Classes/ViewRelated/NUX/ContentViews/Editor/UnifiedPrologueEditorContainerView.swift

```diff
@@ -0,0 +1,19 @@
+import UIKit
+import SwiftUI
+
+/// Prologue editor container view - Needed to bridge SwiftUI to UIKit
+class UnifiedPrologueEditorContainerView: UIView {
```

Comment:
I wonder if we could make this a more generic container view, and we just pass in the SwiftUI contentView we'd like to use with it?